Source/WebInspectorUI/UserInterface/Views/TabBar.css

```diff
@@ -350,40 +347,39 @@ body.window-inactive .tab-bar > .tabs.animating.closing-tab > .item:not(.disable
 }
 
 @media (prefers-color-scheme: dark) {
-    body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar,
-    body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar {
+    body.mac-platform .tab-bar {
```

Comment:
shouldn't this just be `.tab-bar`?  the `:is` and `:not` are mutually exclusive meaning that this would've matched _anything_

Reply:
The `:is()` and `:not()` were mutually exclusive, but they were contributing to being same-level specificity with their light mode counterpart CSS rules. This way they won based on CSS order when in dark mode.

That's also why `body.mac-platform .tab-bar` needs to stay here in the dark mode media query – to be the same specificity as `body.mac-platform .tab-bar` from line 44 for light mode.